backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/aries/config/SchemaService.java

```diff
@@ -150,21 +152,22 @@ SchemaAPI addSchema(@NonNull String schemaId, @Nullable String label,
         return Optional.empty();
     }
 
-    public List<SchemaAPI> listSchemas() {
-        List<SchemaAPI> result = new ArrayList<>();
-        schemaRepo.findAll().forEach(dbS -> result.add(SchemaAPI.from(dbS)));
-        return result;
+    String getDid() {
+        return id.getMyDid() == null ? "" : id.getMyDid();
```

Comment:
I think I will change this in the Identity service. There either is a did or if there is none then we are completely broken anyway. I just kept it that way for testing purposes to be able to start aca-py without a ledger in unit tests.